package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "kuzzle-backoffice",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
I guess we just have a prerelease right now with version 1.1.1, and then include the changes you are working on to the version 1.2.0 ?